lib/entities/achievedMission.dart

```diff
@@ -5,21 +5,30 @@ class AchievedMission {
   int id;
   int attempts;
   bool satelliteUsed;
+  String? comment;
+  int? durationInSeconds;
+
   AchievedMission({
     required this.id,
     required this.attempts,
     required this.satelliteUsed,
+    this.comment,
```

Comment:
t'es sur que c'est bon ?
```suggestion
    this.comment, // avec un commentaire
```